lib/cocoapods/deintegrator.rb

```diff
@@ -40,7 +40,11 @@ def remove_sandbox
     end
 
     def deintegrate_pods_libraries(target)
-      frameworks_build_phase = target.frameworks_build_phase
```

Comment:
calling `frameworks_build_phase` in xcodeproj automatically creates this phase if it does not exist. This can cause diffs to users projects who have targets without it and run `pod install` without integration.